env/netbox.env

```diff
@@ -29,6 +29,6 @@ REDIS_INSECURE_SKIP_TLS_VERIFY=false
 REDIS_PASSWORD=H733Kdjndks81
 REDIS_SSL=false
 RELEASE_CHECK_URL=https://api.github.com/repos/netbox-community/netbox/releases
-SECRET_KEY=r8OwDznj!!dci#P9ghmRfdu1Ysxm0AiPeDCQhKE+N_rClfWNj
+SECRET_KEY=r(m)9nLGnz$(_q3N4z1k(EFsMCjjjzx08x9VhNVcfd%6RF#r!6DE@+V5Zk2X
```

Comment:
Won't this change cause trouble for existing installations?

Reply:
It will invalidate all user sessions and password reset tokens that have been sent but not used.
Netbox will not start with the old key because it is to short, so there is no choice.